chain/chain/src/types.rs

```diff
@@ -36,10 +36,10 @@ use near_primitives::types::{
     StateRoot, StateRootNode,
 };
 use near_primitives::utils::to_timestamp;
-use near_primitives::version::{
-    MIN_GAS_PRICE_NEP_92, MIN_GAS_PRICE_NEP_92_FIX, MIN_PROTOCOL_VERSION_NEP_92,
-    MIN_PROTOCOL_VERSION_NEP_92_FIX, ProtocolVersion,
-};
+use near_primitives::version::{MIN_GAS_PRICE_NEP_92_FIX, ProtocolVersion};
+
+// Define the constant locally since we need to rename it
+const MIN_PROTOCOL_VERSION_NEP_92_FIX: ProtocolVersion = 32;
```

Comment:
Remove the local constant. Instead use the constant from file core/primitives/src/version.rs and rename `_UNUSED_MIN_PROTOCOL_VERSION_NEP_92_FIX` to `MIN_PROTOCOL_VERSION_NEP_92_FIX`